dvm.sh

```diff
@@ -1974,13 +1974,15 @@ export DVM_VERSION="v0.10.0"
     fi
 
     # reset changes if exists
-    git reset --hard HEAD
-    git fetch
-    git checkout "$DVM_LATEST_VERSION"
-
-    dvm_print "DVM has upgrade to latest version, please restart your terminal or run \`source $DVM_PROFILE_FILE\` to apply changes."
+    if git reset --hard HEAD && git fetch --all && git pull origin master --tag && git checkout "$DVM_LATEST_VERSION"
```

Comment:
Not sure the `fetch --all` is necessary

Reply:
`fetch --all` should be enough for this problem, and I keep `pull --tags` to double check

Reply:
The branch to fetch is not in a global variable?

Reply:
No, it is in the global variable `DVM_LATEST_VERSION`